README.md

```diff
@@ -103,6 +103,17 @@ and can contain a regex:
 "uri": "/customer/\\d*/cart"
 ```
 
+To use the option of bypass the mock and issue real http request, the resource files should look like this:
```

Comment:
I think this deserves its own section i.e. 

### Bypassing mocks

If you want to bypass mocks and issue a real http request, add an optional `passThrough` property to your mock.